src/app/components/ItemFilters/ItemFilters.jsx

```diff
@@ -38,10 +39,14 @@ const ItemFilters = (
   // When new items are fetched, update the selected string display.
   useEffect(() => {
     setSelectedFieldDisplayStr(parsedFilterSelections());
-    // Once the new items are fetched, focus on the
-    // filter UI and the results.
-    resultsRef.current && resultsRef.current.focus();
-  }, [numOfFilteredItems, parsedFilterSelections]);
+    // Once the new items are fetched, focus on the filter UI and the
+    // results, but don't do this if the user requested to view all items.
+    // It is annoying if the text keeps getting focused and the page
+    // keeps jumping around.
+    if (!showAll) {
```

Comment:
so where does the brower focus after clicking view all?

Reply:
It doesn't focus anywhere right now. After every 20 new items that are fetched, it would jump to the "X results found". I'll add another flag to focus to the text once fetching is complete.

This is a bit odd because the text doesn't change. If there are 779 items total but only 20 are displayed initially, once the "view all items" button is clicked the text will still say 779 results found. There should be another indicator somewhere that says "loading all items..." and then "loading all items complete". This delves more into the UX side and it'd be good to have a better defined flow.

Perhaps for now, there can be a text underneath the "779 results found" block that displays "loading all items, please wait" and then "loading all items complete" and the focus goes there. What do you think? (we also need better text).

Reply:
maybe a question for clare?

Reply:
Yes. This is updated and once it's on QA we'll ask for a review.